install-dev/upgrade/sql/1.7.8.0.sql

```diff
@@ -39,3 +39,9 @@ ALTER TABLE `PREFIX_log`
   ADD `id_lang` INT(10) NULL DEFAULT NULL after `id_shop_group`,
   ADD `in_all_shops` TINYINT(1) unsigned NOT NULL DEFAULT '0'
 ;
+
+ALTER TABLE `PREFIX_stock_available` MODIFY COLUMN `location` VARCHAR(255) NOT NULL DEFAULT '';
```

Comment:
Why is this one needed? Seems like it is already the current state of this `stock_available.location` column (from the db_struct.sql file)